src/app/system/surveys/surveys.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+    <button mat-raised-button color="primary" [routerLink]="['create']">
+      <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+      Create Surveys
+    </button>
+  </div>
+  
+  <div class="container">
+  
+    <div fxLayout="row">
+      <mat-form-field fxFlex>
+        <mat-label>Filter</mat-label>
+        <input matInput (keyup)="applyFilter($event.target.value)">
+      </mat-form-field>
+    </div>
+  
+    <div class="mat-elevation-z8">
+      <table mat-table [dataSource]="dataSource" matSort>
+        <ng-container matColumnDef="key">
+            <th mat-header-cell *matHeaderCellDef mat-sort-header='key'> Key </th>
+            <td mat-cell *matCellDef="let row"> {{row.key}} </td>
```

Comment:
Change row to survey everywhere. Also, make sure there is a single space in between brackets and content like so: `{{ survey.key }}`.